Sources/Sandbox.Game/Game/Entities/Cube/MySlimBlock.cs

```diff
@@ -193,12 +193,19 @@ public bool HasDeformation
                 return CubeGrid.Skeleton.IsDeformed(Position, 0.0f, CubeGrid, true);
             }
         }
-
+        private int m_lastTotalFrames = 0;
```

Comment:
Adding fields to MySlimBlock makes me nervous - is there any way you could do this that wouldn't increase memory consumption?